source/find_getmore_killcursors_commands.rst

```diff
@@ -365,18 +365,35 @@ More detailed information about the interaction of the **secondaryOk** with **OP
 Behavior of Limit, skip and batchSize
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The new **find** command has different semantics to the existing 3.0 and earlier **OP_QUERY** wire protocol message. The **limit** field is a hard limit on the total number of documents returned by the cursor no matter what **batchSize** is provided.
-
-Once the limit on the cursor has been reached the server will destroy the cursor and return a **cursorId** of **0** in the **OP_REPLY**. This differs from existing **OP_QUERY** behavior where there is no server side concept of limit and where the driver **MUST** keep track of the limit on the client side and **MUST** send a **OP_KILL_CURSORS** wire protocol message when it limit is reached.
-
-When setting the **batchSize** on the **find** and **getMore** command the value MUST be based on the cursor limit calculations specified in the `CRUD`_ specification. 
-
-In the following example the **limit** is set to **4** and the **batchSize** is set to **3** the following commands are executed.
+The new **find** command has different semantics to the existing 3.0 and earlier
+**OP_QUERY** wire protocol message. The **limit** field is a hard limit on the
+total number of documents returned by the cursor no matter what **batchSize** is
+provided. This includes other limiting operations, such as the **$limit**
+aggregation pipeline stage. This differs from existing **OP_QUERY** behavior
+where there is no server side concept of limit and where the driver **MUST**
+keep track of the limit on the client side and **MUST** send a
+**OP_KILL_CURSORS** wire protocol message when it limit is reached.
+
+When setting the **batchSize** on the **find** and **getMore** command the value
```

Comment:
"commands" since you're referring to two types. I'd also suggest writing "the value of limit" since this section refers to "limit, skip, and batchSize". This is the first reference to a value in this paragraph, so I think it's helpful to clarify in case someone immediately jumps into this and starts reading from here.